### DISCUSSION TITLE
Issue with images/bg-images being pulled from WordPress instead of static directory 95% of the time

### DISCUSSION THREAD

**Discussion by bstromegan** [gatsbyjs/gatsby#35534] [Help]
Hoping someone has run into this problem before. 
I'm making a Gatsby site using gatsby-wordpress-source to pull the content from an existing Gutenberg site. The images, which are being created and added to the public folder, aren't being input into the code, which results in images and background images from the content being hoested from the source WP site instead of the Gatsby site.

What's most puzzling about this is that occasionally, one or two images will work and be pulled from the /static/ directory, but it will change on every build, and seems to be inconsistent between local and Gastby Cloud. I know that gatsby-source-wordpres is supposed to be set to generate static images automatically, but I've added the following to gatsby-config.js just in case:
```
resolve: "gatsby-source-wordpress",
      options: {
        url: "https://dev-launchpad-2.pantheonsite.io/graphql",
        html: {
          useGatsbyImage: true,
          createStaticFiles: true,
        },
      },
```
Here are the dependancies in package.json:

```
"dependencies": {
    "gatsby": "^4.13.1",
    "gatsby-awesome-pagination": "^0.3.8",
    "gatsby-background-image": "^1.6.0",
    "gatsby-plugin-gatsby-cloud": "^4.13.0",
    "gatsby-plugin-image": "^2.13.0",
    "gatsby-plugin-sass": "^5.1.1",
    "gatsby-plugin-sharp": "^4.13.0",
    "gatsby-source-filesystem": "^4.13.0",
    "gatsby-source-wordpress": "^6.13.0",
    "gatsby-transformer-sharp": "^4.13.0",
    "html-react-parser": "^1.4.9",
    "node-sass": "^7.0.1",
    "react": "^17.0.1",
    "react-dom": "^17.0.1",
    "sass": "^1.51.0",
    "uuid": "^8.3.2"
  }
```

Content is included as follows:
`      <div dangerouslySetInnerHTML={{ __html: content }} />`

You can see in this example, the first block with a background image (Test Please work) is being pulled from the /static/ directory like we want: https://build-b4cdda9a-53eb-479e-a7d5-5db0021c3a3c.gtsb.io/, but every other image and background image is being pulled from the WP site. 
This following build, where the content got changed, no images are being pulled from the /static/ directory: https://build-a35a3a2c-e592-4da3-acc4-5624ca431a90.gtsb.io/

I've been battling with this for a while and I'm stumped. If anyone has run into this issue and fixed it, or if there's something obvious I may be missing, please let me know.

**Comment by jkgatt** [comment]
I've run into the same issue to be honest, not sure if they're anyone who can help! Ill also find some time to try see whats happening or if there is a better way to print the content!
